### PR TITLE
[Fix] Add BN in FPN to avoid loss Nan in MVXNet

### PR DESCRIPTION
**Pull request by JingweiZhang12**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

https://github.com/open-mmlab/mmdetection3d/issues/2209
https://github.com/open-mmlab/mmdetection3d/issues/1970

## Modification

In order to avoid loss Nan, we add BN with `requires_grad=False` in FPN to make the image features more stable numerically. 
Also, the training precision is almost not affected.

```python
Overall AP11@easy, moderate, hard:
bbox AP11:79.2758, 73.8774, 71.7013
bev  AP11:77.4078, 69.5627, 66.2264
3d   AP11:72.6428, 63.3217, 59.6384
aos  AP11:68.75, 63.20, 61.03

Overall AP40@easy, moderate, hard:
bbox AP40:82.2459, 75.6802, 71.9023
bev  AP40:79.1478, 69.7066, 66.2251
3d   AP40:73.8799, 63.0147, 59.1020
aos  AP40:71.27, 64.48, 61.12
```

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/mvxnet/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ We implement MVX-Net and provide its results and models on KITTI dataset.
...
-| [SECFPN](./dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class.py) | 3 Class | cosine 80e |   6.7    |                | 63.22 | [model](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/mvxnet/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class_20210831_060805-83442923.pth) \| [log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/mvxnet/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class_20210831_060805.log.json) |
+| [SECFPN](./dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class.py) | 3 Class | cosine 40e |   6.7    |                | 63.22 | [model](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/mvxnet/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class_20210831_060805-83442923.pth) \| [log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/mvxnet/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class/dv_mvx-fpn_second_secfpn_adamw_2x8_80e_kitti-3d-3class_20210831_060805.log.json) |
```
Comment (ZwwWayne) [inline configs/mvxnet/README.md:25]: the config name seems to be 80e. also, we should update model log and checkpoint?

**Comment by JingweiZhang12** [comment]
The new log and model are to be uploaded.

**Review by ZwwWayne — APPROVED** [review]
